test/test_networking.py

```diff
@@ -81,3 +82,24 @@ class TestURLs:
     def test_url_ok(self):
         res = networking.url_ok("https://www.gradio.app")
         assert res
+
+
+def test_start_server_app_kwargs():
+    """
+    Test that start_server accepts app_kwargs and they're propagated to FastAPI.
+    """
+    io = Interface(lambda x: x, "number", "number")
```

Comment:
I think it's better to test `launch(app_kwargs=)` directly instead of `networking.start_server()`. Yes it's less of a unit test, but the internals are more of an implementation detail anyways

Reply:
I wrote this test before I'd added this to`launch()` :) Will amend.